docs/reference/TensorFlow.md

```diff
@@ -1,405 +1,394 @@
 
-![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/8mns5yqMfZk/0.jpg)
+    Configure Mbed CLI to find your GNU installation:
 
-[Watch here](http://www.youtube.com/watch?v=8mns5yqMfZk)
+    ```    
+    mbed config -G GCC_ARM_PATH <path_to_your_arm_toolchain>/bin
+    ```
 
-**How to install Ubuntu 20.04 on VirtualBox in Windows 10**
+* Make 3.82 or newer (if you are using an older version, see the troubleshooting section for instructions).
 
-![IMAGE ALT TEXT HERE](http://img.youtube.com/vi/x5MhydijWmc/0.jpg)
+    <span class="notes">**Note**: On a Mac, you might have to use gmake instead of make.</span>
 
-[Watch here](http://www.youtube.com/watch?v=x5MhydijWmc)
+* Python 2.7. We recommend [using pyenv to manage Python versions](https://pypi.org/project/pyenv/).<!--The install page for Mbed CLI now requires 3.7. Why are we asking for 2.7?-->
 
+* Git.
 
-**Getting started**
+* xxd.<!--unless is exists by default on all Linux computers?-->
 
-TensorFlow Lite for Microcontrollers supports several devices out of the box, and is relatively easy to extend to new devices. For this guide, we will focus on the **NXP FRDM K66F**.
+* A development board that can run [TensorFlow Lite for Microcontrollers](https://www.tensorflow.org/lite/microcontrollers#supported_platforms). We tested on the [NXP FRDM-K66F](https://os.mbed.com/platforms/FRDM-K66F/) development board.
 
-![IMAGE ALT TEXT HERE](https://raw.githubusercontent.com/COTASPAR/K66F/master/images/nxp_k66f_board.jpeg)
+* A mini-USB cable.
 
+<!--MarkDown doesn't support embedded YouTube players, which is one reason I removed the videos. Another is that it's better practice to link to official sites, not to videos uploaded by "unofficial" users who may remove them without warning and may, for all we know, be in legal disputes with the companies who created those products (I admit that isn't very likely with "how to install" videos). Also, not sure these videos will work in all countries.-->
+### Download, build and install the application
 
-We will deploy a sample application that uses the microphone on the K66F and a TensorFlow machine learning model to detect the words “yes” and “no”.
+1. Navigate to the directory where you keep code projects.
+1. Download the TensorFlow Lite source code:
 
-To do this, we will show you how to complete the following steps:
+    ```
+    git clone https://github.com/tensorflow/tensorflow.git
+    ```
 
-1.  Download and build the sample application
+1. Navigate into the project directory and build it:
 
+    ```
+    cd tensorflow
 
-2.  Deploy the sample to your K66F
+    make -f tensorflow/lite/micro/tools/make/Makefile TARGET=mbed TAGS="nxp_k66f" generate_micro_speech_mbed_project
+    ```
+    <!--I got a "no such file or directory error". I think it's because there was already `cd tensorflow` and then the path in the command also has `tensorflow`. Removing `tensorflow` from the path removed the error-->
```

Comment:
Okay, just to confirm, the full directory is tensorflow/tensorflow/lite/micro/tools/make/Makefile meaning, you

> git clone https://github.com/tensorflow/tensorflow.git
> cd tensorflow
> make -f tensorflow/lite/micro/tools/make/Makefile TARGET=mbed TAGS="nxp_k66f" generate_micro_speech_mbed_project 

and this should not generate an error. Correct?

Reply:
When I CDed into tensorflow and then also used it in the path, I think it went tensorflow/tensorflow/lite
It worked only when I removed the tensorflow from the path

Reply:
I can't tell you if it actually *works* because Brew won't updated my Make. But it stopped giving the path error.

Reply:
Ok